.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: "🤗 Need help?"
+    url: https://dvc.org/chat
+    about: If you have a question, ask us on Discord. Please join with an invite.
```

```suggestion
    about: If you have a question, ask us on Discord. Please join with this invite 👉 
```
Or maybe use `->` instead of the finger emoji, up to you.

---

Let me check in my fork. :smile:

---

Looks good. Use the `👉`.